src/containers/shared/utils.js

```diff
@@ -157,7 +159,7 @@ export const concatTx = (a, b) => {
       break;
     }
   }
-  return b.splice(0, iterator).concat(a);
+  return a.concat(b.slice(0, iterator));
```

Comment:
Any reason why the order was changed?

Reply:
When using it, it's the same order as `concat`, so it makes more intuitive sense